amundsen_application/static/js/components/NavBar/index.tsx

```diff
@@ -101,7 +114,9 @@ export class NavBar extends React.Component<NavBarProps> {
             </div>
             {this.renderSearchBar()}
             <div id="nav-bar-right" className="ml-auto nav-bar-right">
-              {this.generateNavLinks(AppConfig.navLinks)}
+              {this.generateNavLinks(
+                AppConfig.navLinks.filter(isNavLinkActive)
```

Comment:
nit: One of the reasons I started creating the helpers in config-utils was to abstract away needing to have a deeper understanding of everything in the configuration. What do you think of replacing direct access to `AppConfig` here with some `getNavLinks` helper method, and that method can own the logic for checking what links are supported with current configuration?

Reply:
That makes a lot of sense!
I will add it in this PR soon!